tools/cli/commands-packages.js

```diff
@@ -1649,6 +1649,22 @@ main.registerCommand({
   } else {
     upgradePackageNames = options.args;
   }
+  var upgradePackagesWithoutCordova = _.reject(upgradePackageNames, function (name) {
+    return name.split(':')[0] === 'cordova'; 
+  });
+  if (_.isEqual(upgradePackagesWithoutCordova, upgradePackageNames)) {
+    // There are some cordova packages in the list to update.
+    // We should tell users how to update cordova packages.
+    Console.warn("It appears that you're trying to upgrade a Cordova package.");
+    Console.warn('To perform this action, you should run:');
+    Console.warn('  meteor add cordova:PLUGIN@VERSION');
+    Console.warn('where PLUGIN is your cordova package name and VERSION is, ' +
+                'for example: 1.0.0');
+    Console.warn('To look up the latest version of cordova-plugin-inappbrowser, run:');
+    Console.warn('  npm view PLUGIN version');
+    // Exclude cordova packages
+    upgradePackageNames = upgradePackagesWithoutCordova;
```

Comment:
That would also make more sense.

@laosb I think changes like this are important and helpful, especially from a tech support standpoint. I understand you've had problems running from checkout in the past due to the firewall, and I think you've got some of those checkout problems fixed now, but in the future if you need help confirming simple changes like this and are unable to test them yourself, please make it very clear in the PR that it's not tested and provide instructions.  You know where to find me on chat too.   I'm happy to help!  :wink: